app/styles/_bootstrap.scss

```diff
@@ -829,7 +829,7 @@ pre {
 }
 
 .col-xs-3 {
-    width: 25%;
+    width: 20%;
```

Comment:
@xmudrii changing this style has huge systemwide impacts.  It is part of bootstrap's grid system and should not be changed.  Instead you should change the class used on the columns from col-xs-3 to col-xs-2, or some combination that adds up to 12 (eg if you want one column to be bigger than the others you could make it col-xs-3 and two other columns col-xs-1 as long as the total number added up to 12).  For more details see the bootstrap docs on the grid system here:
http://getbootstrap.com/css/#grid